### PR TITLE
Improve unjinja function

### PR DESCRIPTION
**Pull request by ssbarnea**
Corrects unjinja implementation in order to assure it also removes
statements and comments, not only expressions.

Fixes: #544

### CODE REVIEW

File: lib/ansiblelint/rules/__init__.py [hunk 1]
```diff
@@ -36,7 +36,10 @@ def verbose(self) -> str:
 
     @staticmethod
     def unjinja(text):
-        return re.sub(r"{{[^}]*}}", "JINJA_VAR", text)
+        text = re.sub(r"{{[^}]*}}", "JINJA_EXPRESSION", text)
```
Comment (ssbarnea) [inline lib/ansiblelint/rules/__init__.py:39]: replacement text is not used anywhere, so we can use variations of it.

Comment (albinvass) [inline lib/ansiblelint/rules/__init__.py:39]: Should this not replace jinja looking like "{{ test_var | combine({'key': 'value'}) }}" with "JINJA_EXPRESSION" for some reason?

Because the current regex won't do that.

File: test/TestRuleClass.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+from ansiblelint.rules import AnsibleLintRule
```
Comment (webknjaz) [inline test/TestRuleClass.py:1]: Why did you rename it?

File: lib/ansiblelint/utils.py [hunk 3]
```diff
@@ -225,7 +225,7 @@ def _include_children(basedir, k, v, parent_type):
     return [{'path': result, 'type': parent_type}]
 
 
-def _taskshandlers_children(basedir, k, v, parent_type):
+def _taskshandlers_children(basedir, k, v, parent_type):  # noqa: C901
```
Comment (webknjaz) [inline lib/ansiblelint/utils.py:228]: This and similar changes don't belong here. Use a separate PR for moving ignores from the flake8 config to files.

File: lib/ansiblelint/rules/__init__.py [hunk 4]
```diff
@@ -36,7 +36,10 @@ def verbose(self) -> str:
 
     @staticmethod
     def unjinja(text):
-        return re.sub(r"{{[^}]*}}", "JINJA_VAR", text)
+        text = re.sub(r"{{.+?}}", "JINJA_EXPRESSION", text)
```
Comment (webknjaz) [inline lib/ansiblelint/rules/__init__.py:39]: Is this the only place that is relevant to the PR?

File: test/TestRuleClass.py (hunk 2 above)
Comment (ssbarnea) [reply test/TestRuleClass.py:1]: File contains tests related to base rule class, previous name was referring only to attributes.

File: lib/ansiblelint/rules/__init__.py (hunk 4 above)
Comment (ssbarnea) [reply lib/ansiblelint/rules/__init__.py:39]: Yes.

File: test/TestAnsibleLintRule.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+from ansiblelint.rules import AnsibleLintRule
+
+
+def test_unjinja():
```
Comment (webknjaz) [inline test/TestAnsibleLintRule.py:4]: I'd prefer this to have a docstring but low priority

Comment (webknjaz) [reply test/TestAnsibleLintRule.py:4]: Also, ideally, there could be multiple cases to document more behaviors as separate tests.
